### PR TITLE
PS-6062: Azure Pipelines: task CacheBeta@0 doesn't accept "." (8.0)

### PR DESCRIPTION
**Pull request by inikep**
Use only a major number of compiler version (e.g. "6" instead of "6.0")

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -162,6 +162,8 @@ jobs:
       ccache --print-config
       ccache --zero-stats
 
+      COMPILER_VER=$(CompilerVer)
+      echo '##vso[task.setvariable variable=CompilerMajorVer]'${COMPILER_VER:0:1}
```
Comment (percona-ysorokin) [inline azure-pipelines.yml]: This will make `gcc-1` from `gcc-10` when it is released. Not a problem now but will be in future.

Comment (inikep) [reply azure-pipelines.yml]: Changed to `${COMPILER_VER//[^0-9]/}`. Clang 4.0 will be `40` now.

Comment (inikep) [reply azure-pipelines.yml]: Changed to `${COMPILER_VER%.*}`:
```
$ ver=10.0
$ echo ${ver%.*}
10
```

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Pull request merged**